tests/e2e/vsan_test.go

```diff
@@ -60,28 +63,23 @@ func (s *VsanTestSuite) SetUpSuite(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
-	out, err := dockercli.DeleteVolume(s.config.DockerHosts[0], s.volumeName)
-	c.Assert(err, IsNil, Commentf(out))
+	if s.volumeName != "" {
+		out, err := dockercli.DeleteVolume(s.config.DockerHosts[0], s.volumeName)
+		c.Assert(err, IsNil, Commentf(out))
+	}
 }
 
 var _ = Suite(&VsanTestSuite{})
 
-// Vsan related test
+// Steps:
 // 1. Create a valid vsan policy
-// 2. Create an invalid vsan policy (wrong content)
-// 3. Volume creation with valid policy should pass
-// 4. Valid volume should be accessible
-// 5. Volume creation with non existing policy should fail
-// 6. Volume creation with invalid policy should fail
-func (s *VsanTestSuite) TestVSANPolicy(c *C) {
-	misc.LogTestStart(volCreateTest, "TestVSANPolicy")
-
+// 2. Volume creation with valid policy should pass
+// 3. Valid volume should be accessible
+func (s *VsanTestSuite) TestVolumeWithValidPolicy(c *C) {
+	misc.LogTestStart(vsanTest, "TestVolumeWithValidPolicy")
```

Comment:
as discussed during standup today .. let's use c.TestName and avoid using local constant.

Reference: #1368